src/components/BookItem.js

```diff
@@ -25,4 +38,6 @@ export default function BookItem(props) {
 BookItem.propTypes = {
   id: PropTypes.number.isRequired,
```

Comment:
- [ ] When I tried to delete an item from the list, all the items were deleted and I got this 👇  error message from the console.  Kindly resolve it. ℹ️ 

<img width="794" alt="Untitled" src="https://github.com/andrianarivo/book-store/assets/103386183/77a8eda6-6e5e-47fb-a1a2-af1d5c2d6d60">

Reply:
Thank you